GatherBuddy/Gui/Interface.AlarmTab.cs

```diff
@@ -168,6 +170,7 @@ private void DrawAlarmInfo(ref int alarmIdx, AlarmGroup group)
         var       alarm   = group.Alarms[alarmIdx];
         using var id      = ImRaii.PushId(alarmIdx);
         var       enabled = alarm.Enabled;
+        var       _sounds = new PlaySound(Dalamud.SigScanner);

```

Comment:
Don't instantiate a SeFunction thing more than once, this should be shared between consumers as a parameter.